### PR TITLE
Add missing documentation for class properties

### PR DESCRIPTION
**Pull request by Chouby**
This adds all missing PHPDoc comments to all class properties and enables the sniff Squiz.Commenting.VariableComment in PHPCS config, except for tests.

### CODE REVIEW

File: admin/admin-strings.php [hunk 1]
```diff
@@ -9,8 +9,19 @@
  * @since 1.6
  */
 class PLL_Admin_Strings {
-	protected static $strings = array(); // strings to translate
-	protected static $default_strings; // default strings to register
+	/**
+	 * Stores the strings to translate.
+	 *
+	 * @var array
```
Comment (raaaahman) [inline admin/admin-strings.php:15]: `string[]` is more precise than `array`

File: include/links-permalinks.php [hunk 2]
```diff
@@ -9,9 +9,41 @@
...
+	 * @var bool
+	 */
+	protected $use_trailing_slashes;
+
+	/**
+	 * Rewrite rules to always filter
+	 *
+	 * @var string[]
+	 */
 	protected $always_rewrite = array( 'date', 'root', 'comments', 'search', 'author' );
```
Comment (raaaahman) [inline include/links-permalinks.php:47]: The comment says that these are rules to always _filter_ while the variable name says these are rules to always _rewrite_. Which one is more precise?

File: include/walker-dropdown.php [hunk 3]
```diff
@@ -9,6 +9,12 @@
  * @since 1.2
  */
 class PLL_Walker_Dropdown extends Walker {
+	/**
+	 * Database fields to use.
+	 * See Walker::$db_fields
```
Comment (raaaahman) [inline include/walker-dropdown.php:14]: Maybe use a `@see` doc tag with a link to `Walker::$db_fields`?

File: phpcs.xml.dist [hunk 4]
```diff
@@ -121,6 +119,14 @@
 		<exclude-pattern>*/tests/*</exclude-pattern>
 	</rule>
 
+	<rule ref="Squiz.Commenting.VariableComment.Missing" >
```
Comment (raaaahman) [inline phpcs.xml.dist:122]: The following comment makes me believe that we could skip the variable short description when it doesn't add more information: https://github.com/squizlabs/PHP_CodeSniffer/blob/9bcb29cce831a8f9b2c8aee0bba1a57445f9dbf1/src/Standards/Squiz/Sniffs/Commenting/VariableCommentSniff.php#L83

I'm referring to documentation like:

```php
	/**
	 * Instance of PLL_Model.
	 *
	 * @var PLL_Model
	 */
	public $model;
```

File: settings/table-string.php [hunk 5]
```diff
@@ -14,7 +14,33 @@
...
+
+	/**
+	 * The string groups.
+	 *
+	 * @var string[]
+	 */
+	protected $groups;
+
+	/**
+	 * The selected string group or -1 if none is selected.
```
Comment (raaaahman) [inline settings/table-string.php:39]: Helpful documentation. :+1:

File: admin/admin-strings.php (hunk 1 above)
Comment (Chouby) [reply admin/admin-strings.php:15]: Well in fact it's not an array of strings, but an array of arrays.
See https://github.com/polylang/polylang/blob/20c00ffe116c4f203e5088178c4fede8cd1319e1/admin/admin-strings.php#L48

File: phpcs.xml.dist (hunk 4 above)
Comment (Chouby) [reply phpcs.xml.dist:122]: It seems that you are right. I wonder how it could look in a generated doc.

File: include/walker-dropdown.php (hunk 3 above)
Comment (Chouby) [reply include/walker-dropdown.php:14]: Would
`@see https://developer.wordpress.org/reference/classes/walker/#properties Walker::$db_fields`
be ok for you?

File: include/links-permalinks.php (hunk 2 above)
Comment (Chouby) [reply include/links-permalinks.php:47]: I propose to change to:
`The name of the rewrite rules to always filter.`
This is maybe more explicit.

File: phpcs.xml.dist (hunk 4 above)
Comment (sebastienserre) [reply phpcs.xml.dist:122]: It seems the WP parser doesn't parse the class properties, but I maybe missed something ...

File: include/walker-dropdown.php (hunk 3 above)
Comment (manooweb) [reply include/walker-dropdown.php:14]: Exactly! https://docs.phpdoc.org/3.0/guide/references/phpdoc/tags/see.html#see
Don't forget that you can use a description behind the URL if necessary

**Pull request merged**